core/src/main/java/com/graphhopper/routing/ev/IntEncodedValueImpl.java

```diff
@@ -243,10 +243,11 @@ public final int hashCode() {
 
     @Override
     public int getVersion() {
+        // todonow: there isn't really any 'version' here...
```

Comment:
Do we need some kind of version for EVs? For example this PR changes the format of the serialized strings. Maybe something like `Constants.VERSION_EV`?

Reply:
Actually I renamed it to `Constants.VERSION_EM` now and it represents the format of the stored EM in general, not just for encoded values.

Reply:
getVersion() is gone